mongoengine/document.py

```diff
@@ -531,6 +535,10 @@ def switch_collection(self, collection_name):
         :param str collection_name: The database alias to use for saving the
             document
 
+        :param bool keep_created: keep self._created value after call `swith_db()` when True,
+                    else will always set self._created value to True
+
+
```

Comment:
Here also maybe "keep self._created value after switching collection, else is reset to True"